pkg/engine/jsonContext.go

```diff
@@ -27,6 +27,9 @@ func LoadContext(logger logr.Logger, contextEntries []kyverno.ContextEntry, resC
 	policyName := ctx.Policy.Name
 	if store.GetMock() {
 		rule := store.GetPolicyRuleFromContext(policyName, ruleName)
+		if len(rule.Values) == 0 {
+			return errors.New(fmt.Sprintf("No values found for policy %s rule %s", policyName, ruleName))
```

Comment:
@treydock can you please fix e22-test action.

Reply:
What is it needing to be fixed? On my end I see all Github actions as successful.  I would like to add some e2e tests using policies with variables which is what part of this pull requests fixes.

Reply:
I see the error now, I had to view the pull request diff.  I fixed this issue.